tasks/eth/base-002-fp-upgrade/README.md

```diff
@@ -0,0 +1,42 @@
+# Base Mainnet Fault Proofs Upgrade
+
+Status: [READY TO SIGN]
+
+## Objective
+
+This playbook upgrades Base Mainnet to use the permissionless fault proofs system.
+
+The Base team has already deployed and tested the fault proof contract set. Refer to the [release notes](https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.6.0) for the contract release details.
+
+Governance post of the Fault Proofs upgrade can be found at https://gov.optimism.io/t/upgrade-proposal-10-granite-network-upgrade/8733.
+
+1. Upgrades the `OptimismPortal` proxy implementation.
+2. Upgrades the `SystemConfig` proxy implementation.
+
+## Preparing the Upgrade
+
+The contract implementations and new proxies for the Fault Proof system have been pre-deployed to mainnet.
+
+- `AnchorStateRegistry`: `0x60F1Ea7B3359a4008655df44560B6899B1877a15`,
+- `AnchorStateRegistryProxy`: `0xdB9091e48B1C42992A1213e6916184f9eBDbfEDf`,
+- `DelayedWETH`: `0x71e966ae981d1ce531a7b6d23dc0f27b38409087`
+- `DelayedWETHProxy`: `0xa2f2aC6F5aF72e494A227d79Db20473Cf7A1FFE8`
```

Comment:
There should be two DelayedWETHProxy's, one for the permissionless game and one for the permissioned game. I'd recommend naming them in this list as `DelayedWETHPermissionlessProxy` and `DelayedWETHPermissionedProxy`